tracer/test/Datadog.Trace.Tests/Logging/DatadogLoggingTests.cs

```diff
@@ -205,6 +206,27 @@ public void ErrorsDuringLogging_DontBubbleUp()
             mockLogger.Verify();
         }
 
+        [Fact]
+        public void DuringStartup_OldLogFilesGetDeleted()
+        {
+            var tempLogsDir = Path.Combine(Path.GetTempPath(), "Datadog .NET Tracer\\logs");
+            Directory.CreateDirectory(tempLogsDir);
+
+            for (int i = 0; i < 3; i++)
+            {
+                var logPath = Path.Combine(tempLogsDir, $"dotnet-tracer-{i}.log");
+                File.Create(logPath).Dispose();
+                // Updates LastWriteTime for non 0 indexes files
+                File.SetLastWriteTime(logPath, DateTime.Now.AddDays(-31 * i));
+            }
+
+            DatadogLogging.CleanLogFiles(tempLogsDir);
+            var remainingLogFiles = Directory.GetFiles(tempLogsDir).Length;
+            Directory.Delete(tempLogsDir, true);
+
+            Assert.True(remainingLogFiles == 1);
```

Comment:
I suggest importing the `FluentAssertions` namespace and updating the code to the following. Currently this test is failing on the line but the only diagnostic is that the value was False. This should help with debugging that.

```suggestion
            remainingLogFiles.Should().Be(1);
```

Reply:
Updated test and ran it a couple of times before committing.